kb-api-ws/src/main/java/dk/kb/api/webservice/ApiServiceImpl.java

```diff
@@ -62,7 +67,7 @@ public ApiServiceImpl(){
      * @return
      *          The Solr response in form of String
      */
-    public String getCollectionByQuery(String collection, String qt, String q, List<String> fq, String sort, Integer start, Integer rows, String fl, String df, String wt, Boolean facet, String facetField, String facetPrefix) {
+    public Response getCollectionByQuery(String collection, String qt, String q, List<String> fq, String sort, Integer start, Integer rows, String fl, String df, String wt, Boolean facet, String facetField, String facetPrefix) {
 
         Map<String, String> params = new HashMap<String, String>();
```

Comment:
The `Map` does not allow for duplicate keys, so when the service is called with multiple `fq`, only the last one is preserved. It has to be a `List` or similar structure. For example
```
import org.apache.commons.lang3.tuple.Pair;
...
List<Pair<String, String>> params = new LinkedList<>();
params.add(Pair.of("foo", "bar"));
```

Reply:
I have verified that the change works as intended.